test/quantization/test_quantize_script.py

```diff
@@ -1191,13 +1192,12 @@ def forward(self, x):
 
         data = [(torch.rand((1, 30), dtype=torch.float),
                  torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
-        model = self._test_op_impl(M(), data, "quantized::linear")
-        # make sure there is only one quantize_per_tensor for input
-        # and linear_prepack is folded
-        FileCheck().check_count("aten::quantize_per_tensor", 1, exactly=True) \
-                   .run(model.graph)
-        FileCheck().check_not("quantized::linear_prepack") \
-                   .run(model.graph)
+        model, checker = test_module_graph(
+            M(), data,
+            checks=[['check', ("quantized::linear",)],
+                    ['check_not', ("quantized::linear_prepack",)],
+                    ['check_count', ("aten::quantize_per_tensor", 1, True)]
+                    ])
```

Comment:
I think maybe we can just have three lists, one for check, one for check_not and one for check_count

Reply:
I think you can do two lists, if user want to do check_count they can do it outside of the function.

Reply:
In that case we cannot have an ordered check. Every check will be a separate run.

Reply:
that's OK, I think the purpose of the helper function is to simplify the common cases, we don't have to cover all cases. we can return the model can people can run further checks on them

Reply:
when we cover all cases it becomes too complex and people might as well just call the original checker..